### ISSUE TITLE
PDF footnotes

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#92]
We have a technique for Creating PDF headers and footers, usually inserted
using the source program such as MS Word Headers and Footers. See PDF14.
http://www.w3.org/TR/WCAG20-TECHS/PDF14.html
This causes the screen reader to announce these Headers and footers at the
top and bottom of every page. Sometimes the interrupt reading halfway
through a sentence that continues on the following page. I can imagine this
is quite annoying for screen reader users tp have to listen through a header and a footer between every page.

Visually we just skip over it... Usually most of the information found in the header and footer is available on the home page of the document or some other prominent place.

What is useful is page numbers. However, if the author syncs the page
numbering of the PDF with the document large numbers using PDF17, it makes
me wonder if we should advise the author to mark headers and footers as
artifacts instead...

An organization that teaches PDF, which is fairly well known, and has done
work for Adobe teaches to turn these headers and footers into artifacts so
they will be ignored by a screen reader.

I propose we place a note on PDF14 saying something like this:

 "If the page numbering has been synchronized as in PDF17 and other header
and footer information is available in a prominent place in the document
such as a heading or home page, then headers and footers are not necessary
and can be marked up as artifacts..."

__________
Andrew responds on list

David,

I’m sure that we can debate the right thing to do with header and footer information and still not come to full agreement.  But I agree that repeating “© 2015 MyCorp Industries” or even just the “page 1 of 8” sequence can get tedious for a blind user.

PDF/UA speaks to this:

7.8 Page Headers and footers

Running headers and footers shall be identified as Pagination artifacts and shall be classified as either Header or Footer subtypes as per ISO 32000-1:2008, 14.8.2.2.2, Table 330.

I agree that we need to do something with the PDF14 technique.
________

Katie responds on list
Great catch David!
_______
Jonathan responds
David, there are some good but very specific reasons to not artifact header and footer information.  Otherwise running headers and footer should be made pagination artifacts and the Page Labels feature should be used to add Page numbering including section information to identify pages and sections to assistive technology.  The page label feature is pretty powerful and allows flexibility for different number schemes such as Roman numbers and custom label prefixes.

Some instances of header and footer information that might be useful include classification information on classified documents, copyright information, form numbers, or other content that is not located elsewhere in the document.  In general I recommend marking that content as non-artifact content on the first and last page so it does not break up the reading order of content.
__________

**Comment by DavidMacDonald** [comment]
This will be straight forward, but I'm still not set up with source tree. I hope to have it going in the next couple of weeks. We've come to a good consensus...

**Comment by joshueoconnor** [comment]
@DavidMacDonald Any update on this? Thanks

**Comment by joshueoconnor** [comment]
@DavidMacDonald is working on it

**Comment by awkawk** [comment]
Preparing to mark as deferred due to inactivity and group resource constraints (https://www.w3.org/WAI/GL/wiki/Handling_Issues).
